supertokens_python/recipe/dashboard/interfaces.py

```diff
@@ -98,3 +98,132 @@ def to_json(self) -> Dict[str, Any]:
             "users": users_json,
             "nextPaginationToken": self.next_pagination_token,
         }
+
+
+class DashboardUsersGetResponseWithMetadata(APIResponse):
+    status: str = "OK"
+
+    def __init__(
+        self, users: List[Dict[str, Any]], next_pagination_token: Optional[str]
+    ):
+        self.users = users
+        self.next_pagination_token = next_pagination_token
+
+    def to_json(self) -> Dict[str, Any]:
+        return {
+            "status": self.status,
+            "users": self.users,
+            "nextPaginationToken": self.next_pagination_token,
+        }
+
+
+class UserCountGetAPIResponse(APIResponse):
+    status: str = "OK"
+
+    def __init__(self, count: int):
+        self.count = count
+
+    def to_json(self) -> Dict[str, Any]:
+        return {"status": self.status, "count": self.count}
+
+
+class UserGetAPIOkResponse(APIResponse):
+    status: str = "OK"
+
+    def __init__(self, recipe_id: str, user: Dict[str, Any]):
+        self.recipe_id = recipe_id
+        self.user = user
+
+    def to_json(self) -> Dict[str, Any]:
+        return {
+            "status": self.status,
+            "recipeId": self.recipe_id,
+            "user": self.user,
+        }
+
+
+class UserGetAPINoUserFoundError(APIResponse):
+    status: str = "NO_USER_FOUND_ERROR"
+
+    def to_json(self) -> Dict[str, Any]:
+        return {"status": self.status}
+
+
+class FeatureNotEnabledError(APIResponse):
+    status: str = "FEATURE_NOT_ENABLED_ERROR"
+
+    def to_json(self) -> Dict[str, Any]:
+        return {"status": self.status}
+
+
+class UserMetadataGetAPIOkResponse(APIResponse):
+    status: str = "OK"
+
+    def __init__(self, data: Dict[str, Any]) -> None:
+        self.data = data
+
+    def to_json(self) -> Dict[str, Any]:
+        return {"status": "OK", "data": self.data}
+
+
+class UserSessionsGetAPIResponse(APIResponse):
+    status: str = "OK"
+
+    def __init__(self, sessions: List[SessionInformationResult]):
+        self.sessions = [
+            {
+                "accessTokenPayload": s.access_token_payload,
+                "expiry": s.expiry,
+                "sessionData": s.session_data,
+                "status": "OK",
+                "timeCreated": s.time_created,
+                "userDataInDatabase": {},
```

Comment:
Should not be part of the type

Reply:
Didn't get you.

Reply:
Removed this as it's duplicate of `sessionData`